PlayerInfo/DeviceSettings/PlatformImplementation.cpp

```diff
@@ -118,6 +120,7 @@ class PlayerInfoImplementation : public Exchange::IPlayerProperties, public Exch
         UpdateAudioCodecInfo();
         UpdateVideoCodecInfo();
         Utils::IARM::init();
+        device::Manager::Initialize();
```

Comment:
for upstreaming we would need to wrap this line to compile in  Oout of process mode only , correct?

Reply:
not really, as in process requries initiialization too.

Reply:
actually I tested it "in-process", and fix is required.

Reply:
In case of out process we would need to change the name that is used for iarm bus registration (refer to: https://github.com/LibertyGlobal/rdkservices/pull/16/files#diff-071614dd22144519dbf31723adda4a3d7d947dfe8c39a017272f00d6ec4ccff2R119)